src/main/webapp/index.html

```diff
@@ -43,14 +40,8 @@
           <button class="btn btn-outline-warning my-2 my-sm-0" type="submit">Search</button>
         </form>
         <ul class="navbar-nav mt-2 mt-lg-0">
-          <!-- Sign Up Button -->
-          <li class="nav-item active">
-            <button type="button" class="btn btn-success">Sign Up</button>
-          </li>
-          <!-- Log In Button -->
-          <li class="nav-item">
-            <button type="button" class="btn btn-outline-success">Log In</button>
-          </li>
+          <div id="signup"></div>
+          <div id="login"></div>
```

Comment:
Perhaps an explanation of the function of this two div elements would be helpful!

Reply:
Done!